ASTProcessor/src/test/java/edu/usb/argos/ASTProcessor/AttributeAnalyzerTest/JavaAttributeVisitorTest.java

```diff
@@ -0,0 +1,241 @@
+package edu.usb.argos.ASTProcessor.AttributeAnalyzerTest;
+
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.AttributeHandler;
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInformation;
+import edu.usb.argos.ASTProcessor.visitor.infraestructure.antlr.visitors.method.JavaAttributeVisitor;
+import org.antlr.v4.runtime.CharStream;
+import org.antlr.v4.runtime.CharStreams;
+import org.antlr.v4.runtime.CommonTokenStream;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import edu.usb.argos.ASTProcessor.antlr.JavaLexer;
+import edu.usb.argos.ASTProcessor.antlr.JavaParser;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class JavaAttributeVisitorTest {
+
+    private static JavaAttributeVisitor visitor;
```

Comment:
Why did you put it static? Wouldn't it be better without the static?

Reply:
It's necessary because the composition for use @BeforeAll is with static

Reply:
Ok, thanks a lot for the clarification.